### PR TITLE
KIP-12: KyberDMM Expansion and Liquidity Mining on BSC

### PR DESCRIPTION
**Pull request by ssreelekhya**

### CODE REVIEW

File: KIPs/kip-12.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+kip: 12
+title: KyberDMM Expansion and Liquidity Mining on BSC
+status: Active
+type: Mint KNC
+authors: Kyber Network team
+created: 2021-08-10
+---
+
```
Comment (tranvictor) [inline KIPs/kip-12.md:9]: We need details on proposal parameters such as:
1. start time
2. end time
3. execution
4. executor

Please take a look at other KIPs as a reference

File: KIPs/kip-12.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+kip: 12
+title: KyberDMM Expansion and Liquidity Mining on BSC
+status: Active
+type: Mint KNC
```
Comment (tranvictor) [inline KIPs/kip-12.md:5]: The type of this KIP is not `Mint KNC` as we are not supposed to mint more KNC, we spend KNC from the short executor. Lets change the type to `Spend KNC` instead.

**Review by tranvictor — CHANGES_REQUESTED** [review]

File: KIPs/kip-12.md (hunk 2 above)
Comment (ssreelekhya) [reply KIPs/kip-12.md:5]: Noted

File: KIPs/kip-12.md (hunk 1 above)
Comment (ssreelekhya) [reply KIPs/kip-12.md:9]: Sure,
For now can I just include these

executor: long timelock executor
voting_power: current staked voting power
execution: transfer KNC to TBD
start_timestamp: TBD
end_timestamp: TBD

File: KIPs/kip-12.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+kip: 12
+title: KyberDMM Expansion and Liquidity Mining on BSC
+status: Active
+type: Spend KNC
+authors: Kyber Network team
+created: 2021-08-10
+---
+
+```
+executor: long timelock executor
```
Comment (tranvictor) [inline KIPs/kip-12.md:11]: for spending KNC, we only need short timelock executor

**Review by tranvictor — CHANGES_REQUESTED** [review]

**Review by tranvictor — APPROVED** [review]

**Review by ShaneKyber — APPROVED** [review]

**Review by Anyhowclick — APPROVED** [review]

**Review by loiluu — APPROVED** [review]

**Pull request merged**